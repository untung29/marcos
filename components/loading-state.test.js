import React from "react";
import renderer from "react-test-renderer";
// import { shallow } from "enzyme";
import LoadingState from "./loading-state";

it("<DefaultTextBold />", () => {
  const tree = renderer.create(<LoadingState />).toJSON();
  expect(tree).toMatchSnapshot();
});
