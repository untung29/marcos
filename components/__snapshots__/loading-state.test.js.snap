// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DefaultTextBold /> 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#1B314F",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="small"
  />
  <Text
    style={
      Array [
        Object {
          "color": "white",
          "fontFamily": "PoppinsRegular",
        },
        Object {
          "marginTop": 16,
        },
      ]
    }
  >
    Please wait...
  </Text>
</View>
`;
